src/main/java/frc/robot/commands/TeleopDrive.java

```diff
@@ -15,6 +17,7 @@ public TeleopDrive(){
 	}
 
 	protected void initialize() {
+		teleopLogger.silly("");
```

Comment:
What is this? Should there be a message?

Reply:
Addressed